model/transaction.go

```diff
@@ -134,6 +135,9 @@ func (e *Transaction) Transform(_ context.Context, _ *transform.Config) []beat.E
 			utility.Set(fields, "url", e.Page.URL.Fields())
 		}
 	}
+	if e.Outcome != "" {
```

Comment:
it isn't the case that at `Outcome` is not empty?

Reply:
Yes, good catch. `Outcome` should always be non-empty now.